recipes/wix/meta.yaml

```diff
@@ -0,0 +1,26 @@
+{% set version = "3.10.3" %}
+
+package:
+  name: wix
+  version: {{ version }}
+
+source:
+  path: ./src
```

Comment:
We can, but I have not been able to configure the build correctly on AppVeyor (obscure errors).